### PR TITLE
DB: Fixing some issues

### PR DESCRIPTION
**Pull request by poszer**
##### CHANGES PROPOSED:

-  Quest "The Lost Supplies" : Changed drop of the Scattered Crate drop of Lost Supplies to 100 drop chance.
-  Creature Flanis Swiftwing must be dead.
- Deleted duplicate creature Commander Jordan.
need1creature
shouldbedead

###### ISSUES ADDRESSED:

Closes https://github.com/azerothcore/azerothcore-wotlk/issues/1462

##### TESTS PERFORMED:

Tested in game, everything is fine and correct now.

##### HOW TO TEST THE CHANGES:
1- .go -9979 -3690 22 0
2- Click the box in front of you.

##### KNOWN ISSUES AND TODO LIST:

- [ ]
- [ ] 

##### Target branch(es):

Master

### CODE REVIEW

File: data/sql/updates/pending_db_world/rev_1550318643525669100.sql [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+UPDATE `gameobject_loot_template` SET `Chance`='100' WHERE `Entry`=3597 AND `Item`=6172;
+
+-- Creature "Flanis Swiftwing" should be dead.
+UPDATE `creature_template_addon` SET `emote`=65 WHERE  `entry`=21727;
+UPDATE `creature_template` SET `flags_extra`=2 WHERE  `entry`=21727;
+UPDATE `creature_template` SET `unit_flags`=536904450 WHERE  `entry`=21727;
+UPDATE `creature_template` SET `dynamicflags`=32 WHERE  `entry`=21727;
+
+-- Duplicate creature Commander Jordan
+DELETE FROM `creature` WHERE `guid`=105029;
```
Comment (Nefertumm) [inline data/sql/updates/pending_db_world/rev_1550318643525669100.sql]: You have to delete the creature_addon as well in order to fix travis.

Comment (poszer) [reply data/sql/updates/pending_db_world/rev_1550318643525669100.sql]: Yeah just saw

**Comment by comix1988** [comment]
@poszer 
- I tested 4 "Scattered Crates" and in all I found the quest item
- the NPC Flanis Swiftwing is now dead
- the NPC Commander Jordan is no longer double

tested on ubuntu 18.04.2 LTS

**Comment by poszer** [comment]
> * 
> * the NPC

Nice feedback, thank you so much for testing @comix1988

**Pull request merged**